<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>561</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>301</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>51</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="a_5_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_4_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_3_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_2_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_1_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_1_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_2_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_3_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_4_m_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_5_m_Edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>51</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="a_5_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_4_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_3_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_2_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="a_1_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_1_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_2_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_3_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_4_p_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="b_5_p_Edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>31</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ask5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>ask4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ask3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>ask2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ask1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>buy1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>buy2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>buy3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>buy4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>buy5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>amount</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Insert_Code">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>000002</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Search_Button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="company_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string> Company</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>101</y>
       <width>59</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Up_Prob_Label">
        <property name="text">
         <string>Prob_Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Price_Now</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>pre_close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>high</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>low</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>anmunt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="amount_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>351</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pre_clos_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>176</y>
       <width>69</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="price_now_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>141</y>
       <width>69</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="low_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>281</y>
       <width>69</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="open_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>211</y>
       <width>69</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prob_up_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>106</y>
       <width>69</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="volume_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>316</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="high_Edit">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>246</y>
       <width>69</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="get_top_eight">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Get Top Eight</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="top_1">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top_2">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top_3">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top_4">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="top_5">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top_6">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top_7">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top_8">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>91</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Top1_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Top2_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Top3_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Top4_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>150</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Top5_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Top6_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Top7_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Top8_label">
       <property name="text">
        <string>        --</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionGet_Top_Ten"/>
    <addaction name="actionGet_Last_Ten"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGet_Top_Ten">
   <property name="text">
    <string>Get Top Ten</string>
   </property>
  </action>
  <action name="actionGet_Last_Ten">
   <property name="text">
    <string>Get Last Ten</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
